app/plugin_requests.go

```diff
@@ -111,8 +110,13 @@ func (ch *Channels) ServePluginPublicRequest(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
+	subpath, err := utils.GetSubpathFromConfig(ch.cfgSvc.Config())
+	if err != nil {
+		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
+	}
+
 	publicFilePath := path.Clean(r.URL.Path)
-	prefix := fmt.Sprintf("/plugins/%s/public/", pluginID)
+	prefix := path.Join(subpath, "plugins", pluginID, "public")
```

Comment:
Should [`net/url.JoinPath`](https://pkg.go.dev/net/url@go1.19.4#JoinPath) be used instead of `path.Join` here?

Reply:
Alas, I think the server remains on Go 1.18, precluding the use of this improved API.